src/cli/playbook/ipa.rs

```diff
@@ -158,3 +158,94 @@ where
         breakdowns,
     }
 }
+
+pub async fn playbook_oprf_ipa<F>(
+    mut records: Vec<TestRawDataRecord>,
+    clients: &[MpcHelperClient; 3],
+    query_id: QueryId,
+    query_config: IpaQueryConfig,
+) -> IpaQueryResult
+where
+    F: PrimeField,
+    AdditiveShare<F>: Serializable,
+{
+    let mut buffers: [_; 3] = std::array::from_fn(|_| Vec::new());
+    let query_size = records.len();
+
+    let sz = <OprfReport<Timestamp, BreakdownKey, TriggerValue> as Serializable>::Size::USIZE;
+    for buffer in &mut buffers {
+        buffer.resize(query_size * sz, 0u8);
+    }
+
+    //TODO(richaj) This manual sorting will be removed once we have the PRF sharding in place
+    records.sort_by(|a, b| b.user_id.cmp(&a.user_id));
```

Comment:
Are we also relying on two facts:
(1) The list is expected to be in timestamp sorted order
(2) The `sort_by` used here performs a stable sort

If so, please mention those things in the comment.